File: result/metagpt/claude/humaneval/146.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>

int specialFilter(int* nums, int size) {
    int num = 0;
    for (int i = 0; i < size; i++) {
        if (nums[i] > 10) {
            char buffer[20];
            snprintf(buffer, sizeof(buffer), "%d", nums[i]);
            int len = strlen(buffer);
            if ((buffer[0] - '0') % 2 == 1 && (buffer[len - 1] - '0') % 2 == 1) {
                num++;
            }
        }
    }
    return num;
}

File: result/metagpt/gpt4o/humaneval/146.rs
fn special_filter(nums: &[i32]) -> i32 {
    let mut num = 0;
    for &value in nums.iter() {
        if value > 10 {
            let buffer = value.to_string();
            let len = buffer.len();
            if buffer.chars().next().unwrap().to_digit(10).unwrap() % 2 == 1
                && buffer.chars().last().unwrap().to_digit(10).unwrap() % 2 == 1
            {
                num += 1;
            }
        }
    }
    num
}

fn main() {
    let nums = vec![11, 22, 33, 44, 55];
    let result = special_filter(&nums);
    println!("Number of special filtered elements: {}", result);
}